src/Sarif.Multitool/Sarif.Multitool.csproj

```diff
@@ -2,11 +2,29 @@
   
   <PropertyGroup>
     <OutputType>Exe</OutputType>
+
+    <!-- To Publish single-file exes, run: 
+      dotnet publish Sarif.Multitool.csproj -c Release -f netcoreapp3.0 -r win-x64
+      dotnet publish Sarif.Multitool.csproj -c Release -f netcoreapp3.0 -r linux-x64
+      dotnet publish Sarif.Multitool.csproj -c Release -f netcoreapp3.0 -r osx-x64
+
+      For settings which we want only for these builds, we use [Condition="'$(RuntimeIdentifier)' != ''"] to identify them
+    -->
+
+    <!-- Keep default targets to allow VS2017 use; our build pool is Hosted VS2017 only. -->
     <TargetFrameworks>netcoreapp2.0;netcoreapp2.1</TargetFrameworks>
+    <TargetFrameworks Condition="'$(RuntimeIdentifier)' != ''">netcoreapp2.0;netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
+
+    <!-- When dotnet publish run, produce trimmed single-file exe -->
```

Comment:
>run [](start = 29, length = 3)

"runs"

Reply:
Reworded to be more consistent with the next comment.